tools/topology/topology1/CMakeLists.txt

```diff
@@ -132,6 +132,8 @@ set(TPLGS
 	"sof-tgl-rt711-rt1308\;sof-adl-rt711-4ch\;-DCHANNELS=4\;-DBT_OFFLOAD\;-DDMICPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_48khz.m4\;-DDMIC16KPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_16khz.m4\;-DPLATFORM=adl"
 	"sof-tgl-rt711-rt1308\;sof-adl-rt711\;-DCHANNELS=0\;-DBT_OFFLOAD\;-DPLATFORM=adl"
 	"sof-tgl-rt711-rt1308\;sof-adl-rt711-l0-rt1316-l3-2ch\;-DCHANNELS=2\;-DEXT_AMP\;-DEXT_AMP_REF\;-DAMP_1_LINK=3\;-DDMICPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_48khz.m4\;-DDMIC16KPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_16khz.m4\;-DPLATFORM=adl"
+	"sof-tgl-rt711-rt1308\;sof-rpl-rt711-4ch\;-DCHANNELS=4\;-DBT_OFFLOAD\;-DDMICPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_48khz.m4\;-DDMIC16KPROC_FILTER1=eq_iir_coef_highpass_40hz_20db_16khz.m4\;-DPLATFORM=rpl"
+	"sof-tgl-rt711-rt1308\;sof-rpl-rt711\;-DCHANNELS=0\;-DBT_OFFLOAD\;-DPLATFORM=rpl"
```

Comment:
you also need to change the kernel-dependent part so that we have dynamic pipelines and deep-buffer for tests.

Reply:
@plbossart  Hmm, this exact configuration has been tested with RPL, using sof-adl-rt711-ch4.tplg. I'm not adding or removing anything. If somebody needs other RPL configuration, they can submit patches.

Reply:
Not now read through https://github.com/thesofproject/sof/pull/4611/commits/bb5c6a844a5418de38266e3b3a40d7fe4acbb5bb .... v5.19 won't have RPL support, but I guess we still need these definitions in both places as we don't know which versions distros will pick up.

Reply:
@plbossart Now added the v5.19 definitions.